src/groups/mqb/mqbc/mqbc_clusterfsm.cpp

```diff
@@ -64,6 +64,8 @@ ClusterFSM& ClusterFSM::registerObserver(ClusterFSMObserver* observer)
     // PRECONDITIONS
     BSLS_ASSERT_SAFE(observer);
 
+    BALL_LOG_INFO << "ClusterFSM: Registered 1 new observer.";
```

Comment:
1. It's a very rare operation. We only register once during [`mqbc::ClusterStateManager::start()`](https://github.com/kaikulimu/blazingmq/blob/7a4ca9572260ead058fc638546857023bc6b64be/src/groups/mqb/mqbc/mqbc_clusterstatemanager.cpp#L1378) and unregister once during [`mqbc::ClusterStateManager::stop()`](https://github.com/kaikulimu/blazingmq/blob/7a4ca9572260ead058fc638546857023bc6b64be/src/groups/mqb/mqbc/mqbc_clusterstatemanager.cpp#L1402).
2. Based on our use pattern, it's extremely unlikely to flood logs.
3. DEBUG might be slightly better, will change.
4. Sure, since we are logging on DEBUG level, extending the information makes sense. Will do.

Reply:
Will make the same changes for `mqbc::PartitionFSM::registerObserver()`

Reply:
d